vite.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite'
+import { nodeResolve } from '@rollup/plugin-node-resolve'
+
+export default defineConfig({
+  plugins: [nodeResolve()],
```

Comment:
I asked in the other PR, but why is `nodeResolve` needed?

Reply:
Answered in other PR: https://github.com/melange-re/melange-for-react-devs/pull/16#discussion_r1455989106